types/bull-arena/index.d.ts

```diff
@@ -1,56 +1,80 @@
-// Type definitions for bull-arena 2.6
+// Type definitions for bull-arena 3.0
 // Project: https://github.com/bee-queue/arena/
 // Definitions by: Levi Bostian <https://github.com/levibostian>
+//                 Eli Skeggs <https://github.com/skeggse>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import { RequestHandler } from "express";
-import { ClientOpts } from "redis";
+import BeeQueue = require('bee-queue');
+import BullQueue = require('bull');
```

Comment:
This feels pretty weird to me, but it also seems useful to be able to reference these types explicitly. If this isn't preferable, I could instead do an `interface Queue { new(name: string, settings: any ): any }` but that seems pretty lax.

Reply:
If bee-queue exports types that are likely to conflict with some other package that people use with bull-arena, or is super-complex to compile, you might consider inlining a looser interface like you propose.

Reply:
I just read bull-arena's README, and it looks like almost all users of bull-arena will already have bee-queue installed, so it's perfectly fine for @types/bull-arena to require it as well.

Reply:
FWIW, I don't have bee-queue installed; I do have bull installed.